src/java.base/share/classes/java/util/jar/JarFile.java

```diff
@@ -214,6 +214,11 @@
      */
     public static final String MANIFEST_NAME = META_INF + "MANIFEST.MF";
 
+    /**
+     * The index file name.
+     */
+    public static final String INDEX_NAME = "META-INF/INDEX.LIST";
```

Comment:
Adding this as a public field means it becomes part of the API, so it shouldn't be public here.

Reply:
remove public